From: Maximiliano Curia <maxy@gnuservers.com.ar>
Date: Fri, 15 Jun 2018 13:55:11 +0200
Subject: Clean BUILT_SOURCES

This avoid leaving stale net-connection-editor-resources.{c,h} and wireless-security-resources.{c,h}
---
 panels/network/connection-editor/Makefile.am | 1 +
 panels/network/wireless-security/Makefile.am | 1 +
 2 files changed, 2 insertions(+)

diff --git a/panels/network/connection-editor/Makefile.am b/panels/network/connection-editor/Makefile.am
index 890c23b..98cbddc 100644
--- a/panels/network/connection-editor/Makefile.am
+++ b/panels/network/connection-editor/Makefile.am
@@ -52,6 +52,7 @@ net-connection-editor-resources.c: connection-editor.gresource.xml $(resource_fi
 net-connection-editor-resources.h: connection-editor.gresource.xml $(resource_files)
 	$(AM_V_GEN) glib-compile-resources --target=$@ --sourcedir=$(srcdir) --generate-header --c-name net_connection_editor $<
 
+CLEANFILES = $(BUILT_SOURCES)
 EXTRA_DIST = \
 	$(resource_files) connection-editor.gresource.xml
 
diff --git a/panels/network/wireless-security/Makefile.am b/panels/network/wireless-security/Makefile.am
index 2e8d490..498aaef 100644
--- a/panels/network/wireless-security/Makefile.am
+++ b/panels/network/wireless-security/Makefile.am
@@ -67,6 +67,7 @@ update-from-nma:
 	      git add $(resource_files) && \
 	      git commit -m "network: Update wireless-security UI from network-manager-applet"
 
+CLEANFILES = $(BUILT_SOURCES)
 EXTRA_DIST = $(resource_files) wireless-security.gresource.xml nm-connection-editor-to-network-panel.patch nm-connection-editor-ui-to-network-panel.patch
 
 -include $(top_srcdir)/git.mk
